### DISCUSSION TITLE
mpcie google coral, AMD GPU

### DISCUSSION THREAD

**Discussion by Dvalin21** [AlexeyAB/darknet#8790] [Q&A]
Hello, I'm using darknet feature in a software NVR called Viseron. I wanted to know would it be possible to use a mpcie google coral instead of a Nvidia gpu? And will darknet ever support AMD Gpus? AMD's equivalent is ROCm. Thanks

**Comment by stephanecharette** [comment]
I've been asking for several years whatever happened to the ROCm patches for Darknet.  I've never gotten an answer as to why it was never merged.  See https://www.youtube.com/watch?v=ldumjmPsTck
